chipsec/logger.py

```diff
@@ -345,12 +374,12 @@ def print_banner(self, arguments: Sequence[str]) -> None:
         self.log(f'[CHIPSEC] Arguments: {args}')
         self.log(message)
 
-    def print_banner_properties(self, cs) -> None:
+    def print_banner_properties(self, cs, os_version) -> None:
```

Comment:
can this be changed to pass in the helper string from calling cs.helper.helper.get_info() opposed to expecting a cs object?

Reply:
will look into this for next release